composer.json

```diff
@@ -78,7 +82,13 @@
     "post-create-project-cmd": [
       "@cleanup",
       "WordPressProject\\composer\\ScriptHandler::createRequiredFiles"
+    ],
+    "code-sniff": [
+      "./vendor/bin/phpcs --config-set installed_paths ./vendor/wp-coding-standards/wpcs",
+      "./vendor/bin/phpcs ./web/wp-content"
+    ],
+    "unit-test": [
+      "./vendor/bin/phpunit tests/unit/*"
```

Comment:
I'm ok with that. 

How can we exclude plugins/themes from our tests? Run a test for each plugin folder individually? I could see it getting messy with a log of plugins, especially external ones